ccd-definition/ComplexTypes/GeneratedOrder.json

```diff
@@ -3,8 +3,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "GeneratedOrder",
     "ListElementCode": "type",
-    "FieldType": "FixedRadioList",
-    "FieldTypeParameter": "GeneratedOrderType",
+    "FieldType": "Text",
```

Comment:
reason for change: converted this 'type' from an enum to a String so that I can display the full type of the order (i.e Interim care order) in the tab view

Reply:
is this GeneratedOrder stuff live? I think this would be a breaking change

Reply:
it is live but I checked with RDO, none of our generated order stuff exist in live cases
https://tools.hmcts.net/jira/browse/RDO-5950
(note that it does say 1 'order' exists, but I checked with adam and we're sure it refers to
`"ListElementCode": "Voluntarily in section 20 care order",`
because the query just searched for 'order'. regardless there are two queries, and the second checks for orderCollection (i.e the tab that shows GeneratedOrders), which is empty - so this is safe

Reply:
ok so I guess my next q, why not update the values of the fixed list in ccd?

Reply:
so the fixed list will be used in page 1 (OrderTypeAndDocument) where the user selects the type of order (Blank order, care order, supervision order) - but these orders have subtypes now (interim/final). So before I was just using the same enums which were used in the fixed list to indicate the order 'type' (this 'type' is what is displayed in the tab and not relevant before then). Since I now want to show 'Final care order' or 'Interim supervision order' in one line in the tab, made sense to change this from enum to string - don't want to touch the fixed list as we still need it for selecting the type in OrderTypeAndDocument.

Perhaps it is a bit messy (although I fully blame CCD), but this GeneratedOrder.type is only set when we build the order at the end of the event (so, we're free to customise it, it's essentially just a field that is displayed in the tab. Throughout the journey an order's type is stored in OrderTypeAndDocument). This GeneratedOrder we build at the end is added to the collection of orders, then deleted. makes sense?